app/grants/models.py

```diff
@@ -324,6 +324,16 @@ def updateActiveSubscriptions(self):
         self.activeSubscriptions = handles
 
 
+    @property
+    def recurring_funding_supported(self):
+        return self.contract_version < 2:
```

Comment:
Saw travis was failing on this error - what is this property used for?

Reply:
ill probably thread it through to the recurring fund template if we keep it

Reply:
@owocki this got into grants-round-6 !
I've removed the `:` over here https://github.com/gitcoinco/web/pull/6807/commits/3c9431c83eb406e436d6119f4b6478ae3a645e84

Reply:
derp, my bad